telegram-villa.py

```diff
@@ -1,21 +1,27 @@
 import logging
 from telegram import Update
-from telegram.ext import ApplicationBuilder, ContextTypes, CommandHandler
+from telegram.ext import ApplicationBuilder, MessageHandler, filters
+import os
 
-TOKEN = '6552538823:AAEepUTvV-LEkj0h0Kq5omQuzNk0c0FJEmQ'
+TOKEN = os.getenv("TOKEN")
+APITOKEN = 'JQVl8GNrhQzVhdf7hLVnq1pw6BqvxOGUNhfyBsMm0Ty9H5chvGjYD85m1vhi'
```

Comment:
dont store your token in the code. That's why you have `TOKEN = os.getenv("TOKEN")` already